docs/api-docs/payments/payments-api-overview.md

```diff
@@ -1,88 +1,66 @@
 
 Merchants or shoppers' personal identifiable information (PII) collected by recurring billing apps that consume the BigCommerce Payments API must have its own Privacy Policy sufficient to the requirements of the European Union General Data Protection Requirements (GDPR). The GDPR must be available and displayed to the general public.
 
 
-<div class="HubBlock--callout">
-<div class="CalloutBlock--warning">
-<div class="HubBlock-content">
-
 <!-- theme: warning -->
-### PCI compliance
-> If your application handles credit card data, you will need to be PCI compliant. SAQs (self-assessment questionnaires) can be submitted to
-<a href="mailto:compliance@bigcommerce.com">compliance@bigcommerce.com</a>.
-
-</div>
-</div>
-</div>
+> #### PCI compliance
+> If your application handles credit card data, you will need to be PCI compliant. SAQs (self-assessment questionnaires) can be submitted to <a href="mailto:compliance@bigcommerce.com">compliance@bigcommerce.com</a>.
 
 ## Processing a payment
 
-You can process payments using cards stored with the BigCommerce Stored Credit Cards feature or by providing a credit card number.
+You can process payments charged to either of two main forms of payment: newly-entered credit cards, or stored cards. For a list of supported payment gateways and their feature sets, see [All Available Payment Gateways](https://support.bigcommerce.com/s/article/Available-Payment-Gateways#all-available). 
 
-Attempting to process a payment through the API using the full credit card information may fail if the provider requires 3DS authentication. The card must be saved through a shopper-initiated transaction before it can be charged through the Payments API. For a list of payment gateways that support 3DS, see [All Available Payment Gateways](https://support.bigcommerce.com/s/article/Available-Payment-Gateways#all-available).
-
-<div class="HubBlock--callout">
-<div class="CalloutBlock--info">
-<div class="HubBlock-content">
-
-<!-- theme:  -->
-### Hosted providers
-> The API flow does not currently support hosted/offsite providers, such as PayPal, and wallet type payments, such as Amazon Pay.
-
-</div>
-</div>
-</div>
+<!-- theme: info -->
+> #### Notes
+> * Attempting to process a payment through the API using the full credit card information may fail if the provider requires 3DS authentication. The card must be saved through a shopper-initiated transaction before it can be charged through the Payments API. 
+> * The API flow does not currently support hosted, offsite, or wallet-type providers, such as PayPal and Amazon Pay.
 
 ## Stored cards
```

Comment:
we do support stored paypal accounts via our API. Do we want to update the documentation to include those as well?
<img width="437" alt="image" src="https://user-images.githubusercontent.com/1313863/154868920-0ca8df80-0955-45b5-9fd2-191841928397.png">